docs/getting-started.md

```diff
@@ -15,6 +19,21 @@ You may need to download the [CoreOS Application Signing Key](http://coreos.com/
 
 New releases of `fcct` are backwards compatible with old releases unless otherwise noted.
 
+#### Container
+
+This example uses, but docker can also be used. Substitute v0.2.0 with the desired version.
+
+```bash
+# Pull the desired version
+podman pull quay.io/coreos/fcct:v0.2.0
+
+# Run fcct using standard in and standard out
+podman run -it --rm fcct --pretty --strict < your_config.fcc > transpiled_config.ign
+
+# Run fcct using files. Note that the directory containing the files must be mounted into the container
+podman run --rm -v path_to_config_dir:/srv fcct -input /srv/your_config.fcc -output /srv/transpiled_config.ign --pretty --strict
```

Comment:
While it's still valid we should probably not mix single and double dashes on the parameters.

Reply:
Fixed. Also added a note about the `latest` tag

Reply:
What's the final direction on this? Are you still planning to change this to use the fully-specified image identifier?

Reply:
Do you mean with regard to latest being master or referring to the container as `fcct:version` vs `quay.io/coreos/fcct:version`?

Reply:
The latter (sorry, I misplaced the comment above, it was a followup to @jlebon thread).